sql/core/src/test/resources/sql-tests/results/explain.sql.out

```diff
@@ -58,6 +58,12 @@ struct<plan:string>
 
 (1) Scan parquet default.explain_temp1 
 Output: [key#x, val#x]
+Batched: true
+DataFilters: [isnotnull(key#x), (key#x > 0)]
+Format: Parquet
+Location [not included in comparison]sql/core/spark-warehouse/[not included in comparison]
+PushedFilters: [IsNotNull(key), GreaterThan(key,0)]
```

Comment:
what's the difference between data filters and pushed filters?

Reply:
@cloud-fan Actually i don't know for sure. Looking at the output, could it be one is the catalyst's view of the filter and the other is the datasource's view of the filter i.e after we translate it ? I am guessing here :-)

Reply:
@cloud-fan Checked the code. Our guess was right - fyi
```
private val pushedDownFilters = dataFilters.flatMap(DataSourceStrategy.translateFilter)
  logInfo(s"Pushed Filters: ${pushedDownFilters.mkString(",")}")
```

Reply:
Then I think we only need to mention pushedDownFilters and DPP filters. Other filters are ignored/has no effect.

Reply:
@cloud-fan OK.